LDAR-Sim_inputs.md

```diff
@@ -26,6 +26,263 @@ To support wider use of LDAR-Sim, the University of Calgary and Highwood Emissio
+
+- Third, because methods are often carefully designed and used in treatment / control experiments, it is helpful to allow reuse of specific methods by referring to methods by their `label`.
+
+Consider the following simulation:
+
+```buildoutcfg
+Global parameters
+Programs:
+    Reference program:
+        Reference LDAR method
+    Test program:
+        New LDAR method 1
+    Test program 2:
+        New LDAR method 1
+        New LDAR method 2
+```
+
+Here, there is a situation where `New LDAR method 1` is used in both `Test program` and `Test program 2`. The definition for `New LDAR method 1` can and should be specified only once and reused as it is common among two test programs.
+
+To specify this, we can refer to the program by name in the program definitions with the addition of the `method_labels` parameter to our program. Note, we leave the `methods` key empty as the specified `new_LDAR_method_1` will be injected in at runtime.
+
+```buildoutcfg
+test_program_1 = {
+    'version': '2.0',
+    'parameter_level': 'program',
+    'method_labels': ['new_LDAR_method_1'],
+    'methods': [],
+    .... OTHER PROGRAM PARAMETERS ....
+}
+```
+
+This `new_LDAR_method_1` has to be defined elsewhere in a separate parameter file to be called by name.
+
+```buildoutcfg
+new_LDAR_method_1 = {
+    'version': '2.0',
+    'parameter_level': 'method',
+    'label': 'new_LDAR_method_1',
+    'module': 'OGI',
+    .... OTHER OGI METHOD PARAMETERS ....
+}
+```
+
+When the simulation is put together, the program will be assembled to look like this:
+
+```buildoutcfg
+test_program = {
+    'version': '2.0',
+    'parameter_level': 'program',
+    'method_names': ['new_LDAR_method_1'],
+    'methods': [
+        'new_LDAR_method_1': {
+            'parameter_level': 'method',
+            'label': 'OGI',
+            .... OTHER OGI METHOD PARAMETERS ....
+        }
+    ],
+    .... OTHER PROGRAM PARAMETERS ....
+}
+```
+
+To review, the following parameters are necessary to enable this modularization and reproducibility within the parameter suites:
+
+`parameter_level`: `global`, `program`, or `method`, this defines the target level in the hierarchy. Without specification, LDAR-Sim interprets the parameters as global.
+
+`label`: in method definitions `type` provides a lookup for checking parameters and allows lookup of default parameters, which enables partial parameter specification and out of the box reverse compatibility.
+
+`method_labels`: shorthand method to specify methods by their labels, and include them in more than one simulation easily and reliably. Used only in programs.
```

Comment:
I don't really understand the difference between these two

Reply:
This is a typo. Thanks.